cmd/lora/main.go

```diff
@@ -115,7 +119,24 @@ func main() {
 		}, []string{"method"}),
 	)
 
-	go subscribeToLoRaBroker(svc, mqttConn, logger)
+	msub, err := mqttsub.NewSubscriber(cfg.loraMsgURL, cfg.subTimeout, logger)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Failed to create MQTT subscriber: %s", err))
+		os.Exit(1)
+	}
+
+	go msub.Subscribe(loraServerTopic, func(msg messaging.Message) error {
+		m := lora.Message{}
```

Comment:
I think we have an internal convention to use something like `var m lora.Message` for the declaration.